### PR TITLE
KAFKA-5495: Update docs to use `kafka-consumer-groups.sh` for checking consumer offsets

### PR DESCRIPTION
**Pull request by vahidhashemian**
And remove the deprecated `ConsumerOffsetChecker` example.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5588/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5602/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by ijuma** [comment]
cc @hachikuji

**Comment by vahidhashemian** [comment]
cc @hachikuji

File: docs/ops.html [hunk 1]
```diff
@@ -140,14 +140,29 @@
...
+  Note: This will only show information about consumers that use ZooKeeper (not those using the Java consumer API).
+
+  TOPIC                          PARTITION  CURRENT-OFFSET  LOG-END-OFFSET  LAG        CONSUMER-ID
+  my-topic                       0          2               4               2          my-group_consumer-1
+  my-topic                       1          2               3               1          my-group_consumer-1
+  my-topic                       2          2               3               1          my-group_consumer-2
+  </pre>
 
-  NOTE: Since 0.9.0.0, the kafka.tools.ConsumerOffsetChecker tool has been deprecated. You should use the kafka.admin.ConsumerGroupCommand (or the bin/kafka-consumer-groups.sh script) to manage consumer groups, including consumers created with the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a>.
+  NOTE: Since 0.9.0.0, the old kafka.tools.ConsumerOffsetChecker tool for checking consumer offsets has been deprecated. You should use the kafka.admin.ConsumerGroupCommand (or the bin/kafka-consumer-groups.sh script) to manage consumer groups.
```
Comment (hachikuji) [inline docs/ops.html]: I think we have now removed this tool.

**Review by hachikuji — APPROVED** [review]
One minor comment. The rest LGTM.

File: docs/ops.html (hunk 1 above)
Comment (vahidhashemian) [reply docs/ops.html]: Thanks for catching this. That's correct. I have removed this line in the new commit.

**Pull request closed without merging**